### DISCUSSION TITLE
[자료구조 4일차] 그래프

### DISCUSSION THREAD

**Discussion by seongeun42** [Shortudy-10th/tech-for-developer#33] [오늘의 주제]
그래프 관련 질문 2인조 1개, 3인조 2개씩 총 5개

**Comment by sikutisa** [comment]
* 아래 2가지 질문에 답해주세요.
* 그래프 IS A 트리. 참 거짓여부와 그 이유
* 트리 IS A 그래프. 참 거짓여부와 그 이유

**Comment by sikutisa** [comment]
프로그램상에서 그래프를 구현한다면, 어떤 방법으로 구현할 수 있을까요?

**Comment by watty-exclamationMark** [comment]
대표적인 그래프 탐색법 두 개에 대해 간략하게 말해주세요.

**Comment by watty-exclamationMark** [comment]
완전 그래프와 연결 그래프의 차이점은 무엇일까요?

**Comment by watty-exclamationMark** [comment]
최소신장트리에 대해 설명해주세요.

**Comment by zini9188** [comment]
다익스트라 알고리즘과 벨만포드 알고리즘에 대해 설명하고 두 알고리즘의 차이점을 설명해주세요

**Comment by kjc6735** [comment]
인접 행렬과 인접 리스트의 장단점은 무엇인가요?
